tutorials/polkadot-sdk/system-chains/asset-hub/send-a-tx-paying-fee-different-token.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Send a Transaction While Paying the Fee with a Different Token
+description: This tutorial demonstrates how to send a DOT transfer transaction while paying the fees using a different token on the Asset Hub.
+---
+
+# Send a Transaction While Paying the Fee with a Different Token
+
+## Introduction
+
+The Asset Hub provides a powerful feature that allows users to pay transaction fees using alternative tokens instead of the native token of the chain.
+
+This tutorial demonstrates how to send a DOT transfer transaction while paying the fees using a different token (USDT in this example) on the Asset Hub.
+
+## Environment Setup
+
+Let's set up the development environment for this tutorial:
+
+1. Create a new directory and initialize the project:
```

Comment:
We say "and initialize the project", but that's really what's happening in step 2